src/main/java/command/DeadlineCommand.java

```diff
@@ -0,0 +1,56 @@
+package command;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import exception.DukeException;
+import task.DeadlineTask;
+import task.Task;
+
+public class DeadlineCommand extends Command {
+    private String taskName;
+    private LocalDateTime deadline;
+
+    /**
+     * Constructor for DeadlineCommand
+     *
+     * @param taskName Name of the task
+     * @param deadline deadline of the task
```

Comment:
Perhaps you could standardize if the description of each @param after the parameter name begins with an uppercase or not?
Also, should there be a punctuation behind each parameter description?